### PR TITLE
js error

### PR DESCRIPTION
**Pull request by falcesch**

### CODE REVIEW

File: src/Resources/public/google-places-filter/google-places-filter.js [hunk 1]
```diff
@@ -191,12 +191,12 @@ var GooglePlacesFilter = (function () {
         };
 
         var resetRadiusField = function (place) {
-            if (place && filter.dom.value !== '' && (place.types.includes('street_address') || place.types.includes('route') || place.types.includes('premise'))) {
+            if (place.types && filter.dom.value !== '' && (place.types.includes('street_address') || place.types.includes('route') || place.types.includes('premise'))) {
```
Comment (doishub) [inline src/Resources/public/google-places-filter/google-places-filter.js:194]: If `place` is not existing, from my point of view also here an error is thrown. Could you please add a double check?
```js
if(place && place.types && ...)
// or
if(place?.types && ...) // ie is not supported
```

Comment (falcesch) [reply src/Resources/public/google-places-filter/google-places-filter.js:194]:
```javascript
// or you initialize the function variable
const foo = function (bar=false) {
    console.log(typeof bar);  // boolean
    console.log(typeof bar.sub); // undefined
}
foo();
```

Comment (doishub) [reply src/Resources/public/google-places-filter/google-places-filter.js:194]: Could you please point out a concrete example? I am unsure what exactly you want to achieve with this.

**Comment by doishub** [comment]
Closed due to inactivity.

**Pull request closed without merging**